Makefile

```diff
@@ -12,7 +12,7 @@ test-unit:
 # Note that flake8 reports both W503 and W504 ATM :-/ so ignore W503 for now,
 # which is what should be considered the right setup.
 test-lint:
-	flake8 distgen/ --ignore=W503
+	flake8 distgen/ --ignore=W503 --max-line-length 99
```

Comment:
This is a style change, I have no objection as I'm just a random reviewer :)

Reply:
It is. We can either add this or fix all the lines longer than the default limit. And we can maybe add test-lint to the CI so the future changes won't break it.